de.metas.business/src/main/java/de/metas/bpartner/impexp/BPartnerImportTableSqlUpdater.java

```diff
@@ -640,7 +640,7 @@ private void dbUpdateM_PricingSystems(final String whereClause)
 		int no;
 		sql = new StringBuilder("UPDATE I_BPartner i "
 				+ "SET M_PricingSystem_ID=(SELECT M_PricingSystem_ID FROM M_PricingSystem ps"
-				+ " WHERE i.PricingSystem_Value=ps.value AND ps.AD_Client_ID IN (0, i.AD_Client_ID)) "
+				+ " WHERE i.PricingSystem_Value=ps.value AND ps.AD_Client_ID IN (0, i.AD_Client_ID) and IsActive='Y' ) "
```

Comment:
does this really work? after all, both `i` and `ps` have an `IsActive` column..i would guess

```suggestion
				+ " WHERE i.PricingSystem_Value=ps.value AND ps.AD_Client_ID IN (0, i.AD_Client_ID) and ps.IsActive='Y' ) "
```

Reply:
Yes; weird,  but is working..